TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/automation/AndroidAccessibilitySettingsRetriever.java

```diff
@@ -0,0 +1,42 @@
+package com.novoda.tpbot.automation;
+
+import android.content.ContentResolver;
+import android.provider.Settings;
+import android.util.Log;
+
+public class AndroidAccessibilitySettingsRetriever {
+
+    private static final String EMPTY_STRING = "";
+
+    private final ContentResolver contentResolver;
+
+    public AndroidAccessibilitySettingsRetriever(ContentResolver contentResolver) {
+        this.contentResolver = contentResolver;
+    }
+
+    public String retrieveEnabledAccessibilityServices() {
+        if (canRetrieveAccessibilitySettings()) {
+            return getEnabledAccessibilityServices();
+        }
+        return EMPTY_STRING;
+    }
+
+    private boolean canRetrieveAccessibilitySettings() {
+        try {
+            return Settings.Secure.getInt(
+                    contentResolver,
+                    Settings.Secure.ACCESSIBILITY_ENABLED
+            ) == 1;
```

Comment:
I don't follow how this try matches the exception thrown in catch. In my understanding the `try` is to check whether _any_ accessibility service is enabled. Could do this with `accessibilityManager.isEnabled()`